docs/sources/writing-guide/contribute-release-notes/index.md

```diff
@@ -12,25 +12,130 @@ keywords:
 
 # Contribute to What's New or release notes
 
+This topic explains the decisions and actions associated with collecting, writing, and publishing What's New content.
+
+> **Note:** This topic is only relevant for internal Grafana Labs contributors.
+
+## What's New doc development process
+
+Developing directly in Markdown reduces errors and removes a manual (toilsome and error-prone step), by creating, editing, discussing, and publishing in the same format from which we will publish in GitHub, using the Grafana repo. 
+
+Consider adding an image, though be aware that we will not maintain them, as the document reflect a "point in time." If you need to add an image, refer to [Image, diagram, and screenshot guidelines]({{< relref "../image-guidelines/" >}}).
+
+### When the prior release is complete
+
+Complete the following steps within one week after the prior release goes out.
+
+1. The Docs Squad identifies the technical writer responsible for overseeing the What’s New content development process for the upcoming release.
+1. The identified technical writer cuts a branch and creates a PR with an empty What's New doc to be populated with the What’s New content for the next release. 
+1. The technical writer then pins a link to the [version-number-release-gtm](https://raintank-corp.slack.com/archives/C02L8GQJGBW) Slack channel.
+
+   > **Note:** A Product Manager (PM) is responsible for update the version number of that channel after a release goes out. 
+   
+1. The technical writer posts in the Slack channel that the What's New PR is ready for contributions. 
+1. The PM or PMM on point communicates with eng squads without a PM resource about who’s on point for What’s New PR reviews from within the PM and PMM orgs.
+
+   Refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG) for a list of release dates and go-to-market coordinators.
+
+### Throughout a release
+
+A Grafana release cycle is typically 6-8 weeks, and during that time, developers are empowered to add to the What's New document incrementally. To see a list of release dates and assigned PMs and PMMs, refer to [Grafana Releases and Release Dates](https://grafana-intranet--simpplr.visualforce.com/apex/simpplr__app?u=/site/a145f000001dCXBAA2/page/a125f000001AXrwAAG).
+
+Complete the following steps throughout a release cycle:
+
+1. Dev teams add commits directly against the branch.
+
+   > **Note:** As an alternative, Dev teams can cut PRs against the branch, add What’s New content, collaborate with Reviewers, and merge the PR into the What's New branch.
+
+1. The technical writer periodically checks the What's New PR to determine how much content has been added.
+
+At this point, the larger Marketing org can review the What’s New content in the branch.
+
+### At the end of a release
```

Comment:
I added that the PR is in draft earlier, so maybe somewhere in here there should be something about when to flip it to Ready for review.

Reply:
Good idea. Done.